cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -543,17 +569,17 @@ class fixed_point_column_wrapper : public detail::column_wrapper {
    * // Creates a nullable column of DECIMAL32 elements with 5 elements: {null, 100, null, 300,
    * null}
    * auto elements = make_counting_transform_iterator(0, [](auto i){ return i; });
-   * auto validity = make_counting_transform_iterator(0, [](auto i){ return i%2; });
-   * fixed_point_column_wrapper<int32_t> w(elements, elements + 5, validity, 2);
+   * auto validity = make_counting_transform_iterator(0, [](auto i){ return i % 2; });
```

Comment:
any idea how the old code got past clang-format?

Reply:
it is in comments. don't think it is checked.